### PR TITLE
 #675 Allow gridColor, gridSpacing and gridDotColor

### PR DESCRIPTION
**Pull request by ChrisCanCompute**
gridColor, gridSpacing and gridDotColor can all be set as attributes on a WorkspaceFragment.

(I haven't added backgroundColor, but it should be reasonably straightforward from here if someone else wants to pick that up).

---
This change is [Reviewable](https://reviewable.io/reviews/google/blockly-android/680)

### CODE REVIEW

**Comment by ChrisCanCompute** [comment]
I've just signed the CLA using my new email address.

**Comment by ChrisCanCompute** [comment]
Resubmitted with my personal email address (using my work laptop, but developing on my own time).

File: blocklylib-core/src/main/java/com/google/blockly/android/WorkspaceFragment.java [hunk 1]
```diff
@@ -56,6 +56,9 @@
     private WorkspaceView mWorkspaceView;
 
     private boolean mDrawGrid = true;
+    private Integer mGridColor = null;
```
Comment (AnmAtAnm) [inline blocklylib-core/src/main/java/com/google/blockly/android/WorkspaceFragment.java]: We can make these unboxed ints if we just refer to the defaults in WorkspaceGridRenderer.  Make the constants package scoped to make them visible.  That also allows us to sidestep the if statements (with a tiny amount of one-time over overhead).

Comment (ChrisCanCompute) [reply blocklylib-core/src/main/java/com/google/blockly/android/WorkspaceFragment.java]: WorkspaceFragment and WorkspaceGridRenderer are in different packages. I can either move the defaults to the former or make the latter public.
Any preference?

Comment (ChrisCanCompute) [reply blocklylib-core/src/main/java/com/google/blockly/android/WorkspaceFragment.java]: I've gone for making WorkspaceGridRenderer public, but happy to change it if you'd prefer.

**Comment by AnmAtAnm** [comment]
Awesome.  Thanks again, Chris!

**Pull request merged**